SU2_CFD/include/numerics/flow/flow_sources.hpp

```diff
@@ -153,7 +153,7 @@ class CSourceIncAxisymmetric_Flow final : public CSourceBase_Flow {
  * \author T. Economon
  */
 class CSourceBodyForce final : public CSourceBase_Flow {
-  su2double Body_Force_Vector[3];
+  vector<su2double> Body_Force_Vector;
```

Comment:
No go :) we want compile-time sizes whenever possible/practical for performance. The hands of the compiler are basically tied if data is coming/going to the heap, it cannot reorder loads/stores for example.